scooter_manipulation/launch/ur_driver_bringup.launch.py

```diff
@@ -0,0 +1,618 @@
+import os
```

Comment:
Is this the driver from UR? I'll be honest not sure what this is doing. What's happening with the MongoDB instance?

Reply:
@sLaderoute merged this from the two launch files we need to start the drivers; everything here aside from the stuff we made is necessary to start the UR5 driver with MoveIt support (even the MongoDB server!) The nodes in `scooter_manipulation` are launched here so that we can pass the robot description as I have elaborated upon in Discord.